.github/workflows/dependencies/dependencies_cuda.sh

```diff
@@ -24,4 +24,4 @@ sudo apt-get install -y \
 
 export PATH=/usr/local/nvidia/bin:/usr/local/cuda-12.6/bin:${PATH}
 which nvcc
-nvcc --version
+nvcc --version
```

Comment:
Please try to avoid committing extraneous changes like these when submitting a PR.

Reply:
Removed the newline.